src/uthash.h

```diff
@@ -255,34 +255,34 @@ do {
 #define HASH_GHV_COMPUTE(hh, keyptr, keylen_in, add)                             \
   HASH_FCN((keyptr), (keylen_in), (add)->hh.hashv)
 
-#define HASH_ADD_SORTED(hh,head,keyptr,keylen_in,hashval,add,cmpfcn)             \
+#define HASH_SORTED_ADD(hh,head,keyptr,keylen_in,hashval,add,cmpfcn)             \
```

Comment:
Oops, I misunderstood (as I myself predicted only days ago :D) — this macro isn't part of the public API.